--- a/willuslib/filelist.c
+++ b/willuslib/filelist.c
@@ -1103,7 +1103,9 @@
     sprintf(cmd,"zip \"%s\" -@ < \"%s\" 1> z1.out 2> z1.err",zipfileabs,tmpfile);
 #else
     sprintf(cmd,"zip \"%s\" -@ < \"%s\" > /dev/null",zipfileabs,tmpfile);
 #endif
+#ifndef __IPHONE_OS_VERSION_MIN_REQUIRED
     system(cmd);
+#endif
     wfile_set_wd(curdir);
     wfile_remove_utf8(tmpfile);
     if (wfile_status(zipfileabs)!=1)
